docs/pages/guides/pricing.mdx

```diff
@@ -191,6 +181,31 @@ export default async function auth(req, res) {
 }
 ```
 
+## Testing use
+
+If you connect rooms and call the Liveblocks API during any automated testing,
```

Comment:
```suggestion
If you connect to rooms and call the Liveblocks API during any automated testing,
```

Reply:
I think above makes more senese, aside from that good to go!

Reply:
It's much better, thank you @taylorpwhipp !